data.sql

```diff
@@ -1,5 +1,13 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+ VALUES ('Agumon', '2020-02-03', 0, 'true', 10.23),
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+ VALUES ('Gabumon', '2018-11-15', 2, 'true', 8),
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg) 
+VALUES ('Pikachu', '2021-01-07', 1, 'false', 15.04),
+
```

Comment:
- [ ] Kindly use the _**semi-colon (;)**_ instead of the _**comma (,)**_ at the end of every statement written in PSQL. Using the **_comma (,)_** sign will cause a syntax error. 👍 😎 

![image](https://user-images.githubusercontent.com/77106962/228661007-10e07185-a36a-44af-871e-4ef49cc3039c.png)